src/SignalR/clients/ts/signalr/src/HubConnectionBuilder.ts

```diff
@@ -44,15 +69,17 @@ export class HubConnectionBuilder {
 
     /** Configures custom logging for the {@link @aspnet/signalr.HubConnection}.
      *
-     * @param {LogLevel | ILogger} logging An object implementing the {@link @aspnet/signalr.ILogger} interface or {@link @aspnet/signalr.LogLevel}.
+     * @param {LogLevel | string | ILogger} logging A {@link @aspnet/signalr.LogLevel}, a string representing a LogLevel, or an object implementing the {@link @aspnet/signalr.ILogger} interface. See {@link https://docs.microsoft.com/en-us/aspnet/core/signalr/configuration#configure-logging|the documentation for client logging configuration} for more details.
      * @returns The {@link @aspnet/signalr.HubConnectionBuilder} instance, for chaining.
      */
-    public configureLogging(logging: LogLevel | ILogger): HubConnectionBuilder;
-    public configureLogging(logging: LogLevel | ILogger): HubConnectionBuilder {
+    public configureLogging(logging: LogLevel | string | ILogger): HubConnectionBuilder {
```

Comment:
Should the argument here also be `keyof typeof  LogLevelNameMapping` ?

Reply:
Ah no. realized as soon I I wrote it. It's kind of the point of the pr

Reply:
That variable isn't exported from `index` so consumers won't be able to see it.